octobot_trading/enums.py

```diff
@@ -101,6 +101,24 @@ class PositionStatus(enum.Enum):
     ADL = "auto_deleveraging"
 
 
+class BlockchainTransactionStatus(enum.Enum):
+    CREATED = "created"
+    CONFIRMING = "confirming"
+    REPLACED = "replaced"
+    FAIL = "fail"
+    SUCCESS = "success"
+
+
+class TransactionType(enum.Enum):
+    BLOCKCHAIN_DEPOSIT = "blockchain_deposit"
+    BLOCKCHAIN_WITHDRAW = "blockchain_withdraw"
```

Comment:
```suggestion
    BLOCKCHAIN_WITHDRAWAL = "blockchain_withdrawal"
```
(source: https://thecontentauthority.com/blog/withdraw-vs-withdrawal)